glide.lock

```diff
@@ -30,6 +30,10 @@ imports:
   version: 9fa818a44c2bf1396a17f9d5a3c0f6dd39d2ff8e
 - name: github.com/hashicorp/go-cleanhttp
   version: ad28ea4487f05916463e2423a55166280e8254b5
+- name: github.com/hashicorp/go-version
+  version: fc61389e27c71d120f87031ca8c88a3428f372dd
+- name: github.com/looplab/logspout-logstash
```

Comment:
i don't think we don't want this here

Reply:
I added the go-version lib to do the api version compare.

I think the logspout-logstash was added by glide since its in custom/modules.go file.  I'm not an expert on glide. How do I tell glide not to include that when I do glide get/install?

Reply:
👍 on the `go-version` lib

go ahead and manually remove L35-L36 here